src/main/java/frc/robot/subsystems/climberSubsystem/ClimberCommandFactory.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.climberSubsystem;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.Constants.ClimberConstants;
+import java.util.function.BooleanSupplier;
+import java.util.function.Consumer;
+
+public class ClimberCommandFactory {
+  private final ClimberSubsystem subsystem;
+  private final Runnable noop = () -> {};
+
+  public ClimberCommandFactory(ClimberSubsystem subsystem) {
+    this.subsystem = subsystem;
+  }
+
+  public Command raise() {
+    if (subsystem == null) return new InstantCommand();
+    Timer minTime = new Timer();
+    return new FunctionalCommand(
+        () -> {
+          subsystem.applyCurrentDemand(10);
+          minTime.start();
+        },
+        noop,
+        (interupt) -> {
+          if (interupt) return;
```

Comment:
This doesn't turn off the motors if the command was interrupted?

nit:  typo

Reply:
it doesn't take stop the motors because the other command is taking responsibility for the climber

Reply:
Add code comments of the intent then, since its uncommon for the interrupt behavior to be different than the non-interrupt behavior.